.circleci/config.yml

```diff
@@ -93,24 +93,11 @@ jobs:
       - install-deps
       - prepare
       - run:
-          command: make build
-      - run:
-          name: Install golangci-lint
-          command: >
-            (
-              cd $(mktemp -d);
-              go mod init a;
-              go get github.com/golangci/golangci-lint/cmd/golangci-lint@v<< parameters.golangci-lint-version >>
-              go build github.com/golangci/golangci-lint/cmd/golangci-lint
-              mkdir -p $HOME/.local/bin
-              mv -f golangci-lint $HOME/.local/bin/
-            )
-      - run:
-          command: make .nomaprange.so
+          command: make -j3 support/tools/golangci-lint support/tools/no-map-range.so
```

Comment:
`-j$(nproc)`?

Reply:
I think circle reports way more cores than it really gives you.

Reply:
Fair enough.